main.tf

```diff
@@ -231,6 +231,20 @@ resource "google_compute_address" "static_nginx_ip" {
   }
 }
 
+resource "helm_release" "nginx_controller" {
+  count            = var.nginx_controller.enabled ? 1 : 0
+  name             = "ingress-nginx"
```

Comment:
Did not mention as `ingress-nginx-controller` as the helm_release adds `-controller` while creation